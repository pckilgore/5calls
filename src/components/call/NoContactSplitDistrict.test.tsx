import * as React from 'react';
import i18n from '../../services/i18n';
import { shallow } from 'enzyme';
import { I18nextProvider } from 'react-i18next';
import { NoContactSplitDistrict } from './';

test('NoContactSplitDistrict component snapshot renders correctly when splitDistrict=true', () => {
  const component = shallow(
    <I18nextProvider i18n={i18n}>
      <NoContactSplitDistrict splitDistrict={false} />
    </I18nextProvider>
  );
  expect(component).toMatchSnapshot();
});
