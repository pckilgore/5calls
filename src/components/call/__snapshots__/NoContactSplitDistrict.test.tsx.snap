// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContactSplitDistrict component snapshot renders correctly when splitDistrict=true 1`] = `
<Translate(NoContactSplitDistrict)
  splitDistrict={false}
/>
`;
